python/cuml/tsa/arima.pyx

```diff
@@ -103,7 +106,7 @@ cdef extern from "cuml/tsa/batched_arima.hpp" namespace "ML":
 
     void estimate_x0(
         handle_t& handle, ARIMAParams[double]& params, const double* d_y,
-        int batch_size, int nobs, const ARIMAOrder& order)
+        int batch_size, int nobs, const ARIMAOrder& order, bool missing)
```

Comment:
Commenting here because I cannot add the comment to line 155: Please correct the docstring regarding missing values.

> This class can fit an ARIMA(p,d,q) or ARIMA(p,d,q)(P,D,Q)_s model to a
>    batch of time series of the same length with no missing values.